src/robot/running/arguments/typeconverter.py

```diff
@@ -14,6 +14,7 @@
 #  limitations under the License.
 
 from ast import literal_eval
+from enum import EnumMeta
```

Comment:
Enums being new in Python 3.4, this is probably the first change not compatible with Python 3.3. Good that we decided to drop its support along with Python 2.6 in #2276.